management/connection.go

```diff
@@ -1346,3 +1441,88 @@ func (m *ConnectionManager) ReadByName(ctx context.Context, name string, opts ..
 	}
 	return nil, &managementError{404, "Not Found", "Connection not found"}
 }
+
+// CreateSCIMConfiguration creates a SCIM configuration for a connection by its connection ID.
+//
+// Note: This method only works with the following enterprise connections:
+//   - Authentication > Enterprise > SAML
+//   - Authentication > Enterprise > OpenID Connect
+//   - Authentication > Enterprise > Okta Workforce
+//   - Authentication > Enterprise > Microsoft Azure AD
+//
+// Parameters:
+//   - scimConfig (optional): The SCIM configuration details. Only `mapping` and `user_id_attribute` fields are used.
+//     This parameter can be passed as nil or empty.
+//
+// `mapping`: Specifies a mapping between SCIM protocol user schema and Auth0 user schema.
+// If not provided, a default mapping based on the connection type (e.g., Okta, SAML) will be used.
+//
+// `user_id_attribute`: Specifies the SCIM attribute containing the unique user identifier
+// presented in the SAML assertion or ID token during user login. If not provided, it defaults to
+// `userName` for SAML connections and `externalId` for OIDC connections.
+//
+// For more details, see: https://auth0.com/docs/api/management/v2/connections/post-scim-configuration
+func (m *ConnectionManager) CreateSCIMConfiguration(ctx context.Context, id string, scimConfig *SCIMConfiguration, opts ...RequestOption) error {
```

Comment:
Should we return the SCIMConfiguration that is returned by the POST endpoint?

Reply:
No, we don't need to return anything because the value in scimConfig is updated directly after getting the results from the API. We're using a pointer to the configuration, which ensures that the response updates the pointer's value.